src/sentry/models/outbox.py

```diff
@@ -101,6 +101,8 @@ class OutboxCategory(IntEnum):
     AUTH_PROVIDER_UPDATE = 24
     AUTH_IDENTITY_UPDATE = 25
 
+    PARTNER_ACCOUNT_UPDATE = 26
```

Comment:
Do we need a new outbox scope for this as well?

Reply:
I placed this in the `ORGANIZATION_SCOPE` based on my skimming of https://github.com/getsentry/getsentry/pull/11630/.

Reply:
I think it'd make sense to throw it under the `PROVISION_SCOPE` since it's usually hit in those flows anyway.

Reply:
:+1: provision makes sense to me, as partner updates have to be sequenced with other provisioning operations.